public/app/features/dashboard/services/TimeSrv.ts

```diff
@@ -264,7 +264,7 @@ export class TimeSrv {
     this.$timeout(this.refreshDashboard.bind(this), 0);
   }
 
-  timeRangeForUrl() {
+  timeRangeForUrl = () => {
```

Comment:
nit: is this a deliberate change?

Reply:
yeah because of the location utility initialisation which made this this function loose scope